test/signing_test.go

```diff
@@ -8,15 +8,420 @@ import (
 
 	"github.com/Azure/dalec"
 	"github.com/Azure/dalec/test/testenv"
+	"github.com/moby/buildkit/client/llb"
 	gwclient "github.com/moby/buildkit/frontend/gateway/client"
 	"github.com/stretchr/testify/assert"
 )
 
-func distroSigningTest(t *testing.T, spec *dalec.Spec, buildTarget string) testenv.TestFunc {
+func runTest(t *testing.T, f testenv.TestFunc) {
+	t.Helper()
+	ctx := startTestSpan(baseCtx, t)
+	testEnv.RunTest(ctx, t, f)
+}
+
+func newSpec() *dalec.Spec {
```

Comment:
This is a fairly generic function name that's only minimally useful.
`newSimpleSpec`?

Reply:
done